util/chplenv/chpl_llvm.py

```diff
@@ -211,12 +211,22 @@ def get_llvm_clang(lang):
     llvm_val = get()
     if llvm_val == 'system':
         bindir = get_system_llvm_config_bindir()
-        return os.path.join(bindir, clang_name)
+        clang = os.path.join(bindir, clang_name)
     elif llvm_val == 'bundled':
         llvm_subdir = get_bundled_llvm_dir()
-        return os.path.join(llvm_subdir, 'bin', clang_name)
+        clang = os.path.join(llvm_subdir, 'bin', clang_name)
     else:
         return ''
+    # tack on the contents of configured-clang-sysroot-arguments
+    llvminstall = get_chpl_third_party() + "/llvm/install/" + get_uniq_cfg_path_for(llvm_val)
```

Comment:
I don't think it really matters (because we aren't near Windows support and even there `/` is recently allowed as a path separator) but it's arguably more normal Python style to use `os.path.join`

Reply:
So *that's* what that means.  For some reason `join` isn't all that obvious a verb to me, though I admittedly like it better than `concat` now that I understand it (and I think the `os` always made me think that something deep and systems-y other than simple string manipulation was going on here).  I folded this into the os.path.join on the next line.